sgbackend/mail.py

```diff
@@ -51,67 +44,57 @@ def send_messages(self, emails):
 
         count = 0
         for email in emails:
-            try:
-                self.sg.send(self._build_sg_mail(email))
-                count += 1
-            except sendgrid.SendGridClientError:
-                if not self.fail_silently:
-                    raise
-            except sendgrid.SendGridServerError:
-                if not self.fail_silently:
-                    raise
+            mail = self._build_sg_mail(email)
+            response = self.sg.client.mail.send.post(request_body=mail)
+            count += 1
```

Comment:
Yes, for now you need to catch the errors like this: https://github.com/sendgrid/sendgrid-python/blob/master/TROUBLESHOOTING.md#error-messages